Synthesis.Bethesda.GUI/Views/Initialization/SolutionInitView.xaml

```diff
@@ -119,10 +121,29 @@
                 </Grid>
             </TabItem>
         </TabControl>
-        <CheckBox Grid.Row="1"
-            x:Name="OpenVsAfter"
-            Margin="10,15,0,6"
-            Content="Open Visual Studio After Creating" />
+        <Grid Grid.Row="1" Margin="0,15,0,6">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <TextBlock Grid.Row="0" Margin="0 4" Text="Open With"/>
+            <ComboBox x:Name="OpenWithComboBox" Grid.Row="1">
+                <ComboBox.ItemsSource>
+                    <x:Array Type="{x:Type sys:Enum}">
+                        <gui:OpenWithEnum>None</gui:OpenWithEnum>
+                        <gui:OpenWithEnum>SystemDefault</gui:OpenWithEnum>
+                        <gui:OpenWithEnum>VisualStudio</gui:OpenWithEnum>
+                        <gui:OpenWithEnum>Rider</gui:OpenWithEnum>
+                        <gui:OpenWithEnum>VisualCode</gui:OpenWithEnum>
```

Comment:
Might consider swapping to something like this so it automatically adapts with enum changes:
https://github.com/Noggog/Synthesis/blob/1cb192e311410899c9818bd5318d7c9cfeb7d10a/Synthesis.Bethesda.GUI/ViewModels/Profiles/NewProfileVM.cs#L26